src/Form/QuestionType/QuestionTypeRequestType.php

```diff
@@ -0,0 +1,141 @@
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Form\QuestionType;
+
+use Glpi\Application\View\TemplateRenderer;
+use Glpi\Form\Question;
+use Override;
+use Ticket;
+
+/**
+ * Long answers are multiple lines inputs used to answer questions with as much details as needed.
+ */
+final class QuestionTypeRequestType extends AbstractQuestionType
+{
+    /**
+     * Retrieve the default value for the request type question type
+     *
+     * @param Question|null $question The question to retrieve the default value from
+     * @return int
+     */
+    public function getDefaultValue(?Question $question): int
+    {
+        if (
+            $question !== null
+            && isset($question->fields['default_value'])
+        ) {
+            if (is_int($question->fields['default_value'])) {
+                return $question->fields['default_value'];
+            } elseif (is_numeric($question->fields['default_value'])) {
+                return (int) $question->fields['default_value'];
+            }
+        }
+
+        return 0;
+    }
+
+    #[Override]
+    public function renderAdministrationTemplate(?Question $question): string
+    {
+        $template = <<<TWIG
+            {% import 'components/form/fields_macros.html.twig' as fields %}
+
+            {{ fields.dropdownArrayField(
+                'default_value',
+                value,
+                request_types,
+                '',
+                {
+                    'init'               : init,
+                    'no_label'           : true,
+                    'display_emptychoice': true
+                }
+            ) }}
+TWIG;
+
+        $twig = TemplateRenderer::getInstance();
+        return $twig->renderFromStringTemplate($template, [
+            'init'               => $question != null,
+            'value'              => $this->getDefaultValue($question),
+            'request_types'     => Ticket::getTypes()
+        ]);
+    }
+
+    #[Override]
+    public function renderEndUserTemplate(Question $question): string
+    {
+        $template = <<<TWIG
+        {% import 'components/form/fields_macros.html.twig' as fields %}
+
+        {{ fields.dropdownArrayField(
+            'answers[' ~ question.fields.id ~ ']',
```

Comment:
I think this must be replaced by the new format after you rebase the PR to the latest changes